recipes/mpfr/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.3.0)
+project(test_package C)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} ../test_package/test_package.c)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::mpfr)
```

```suggestion

find_package(mpfr CONFIG REQUIRED)

add_executable(${PROJECT_NAME} ../test_package/test_package.c)
target_link_libraries(${PROJECT_NAME} mpfr::mpfr)
```

So you can validate `cmake_find_package_multi`, plus, `CONAN_PKG` is basically deprecated. Or, just accept SpaceIm suggestion, which re-uses test_package implementation.